Tests/IntegrationTests/Fixtures/1Dimension/createNewNodeInContainer.e2e.js

```diff
@@ -0,0 +1,38 @@
+import {Selector, RequestLogger} from 'testcafe';
+import {ReactSelector} from 'testcafe-react-selectors';
+import {beforeEach, subSection, checkPropTypes} from './../../utils.js';
+import {Page} from './../../pageModel';
+
+/* global fixture:true */
+
+const changeRequestLogger = RequestLogger(request => request.url.endsWith('/neos/ui-services/change') && request.method === 'post' && request.isAjax);
```

Comment:
Why are you adding a request logger if you are not using it anywhere?

Reply:
Copy & paste, I'll remove it.
Btw. thanks to your hint the test was done quite quickly.